### PR TITLE
API Product Invoke Test

### PR DESCRIPTION
**Pull request by Tharanidk**

### CODE REVIEW

File: portals/devportal/package.json [hunk 1]
```diff
@@ -17,7 +17,8 @@
         "lint": "eslint --ignore-pattern '*.test.js' --ignore-pattern '*.test.jsx' --quiet -c .eslintrc.js --ext .jsx,.js source",
         "i18n": "extract-messages -l=en -o site//public//locales// -d en --extractFromFormatMessageCall --flat 'source//src//app//**//*.jsx'",
         "check-unused": "webpack --mode production --env.unused true --display errors-only",
-        "start": "webpack serve --compress --https --port 8084 --mode development "
+        "start": "webpack serve --compress --https --port 8084 --mode development ",
+        "replace": "ls | entr -s 'rimraf /home/tharani/clone/pack/repository/deployment/server/jaggeryapps/devportal/site/public/dist && cp -R /home/tharani/clone/apim-apps/portals/devportal/site/public/dist /home/tharani/clone/pack/repository/deployment/server/jaggeryapps/devportal/site/public ' "
```
Comment (madushadhanushka) [inline portals/devportal/package.json:21]: Shall we remove these two lines?

Comment (Tharanidk) [reply portals/devportal/package.json]: Done Changes

File: portals/publisher/package.json [hunk 2]
```diff
@@ -19,7 +19,8 @@
         "i18n:compile": "formatjs compile site/public/locales/raw.en.json --ast --out-file site/public/locales/en.json",
         "start": "webpack serve --compress --https --port 8081 --mode development",
         "start:mocked": "concurrently npm:mocked 'WSO2_UI_MOCKED=true webpack serve --compress --https --port 8081 --mode production --progress'",
-        "mocked": "multi-mock-oas"
+        "mocked": "multi-mock-oas",
+        "replace": "ls | entr -s 'rimraf /home/tharani/clone/pack/repository/deployment/server/jaggeryapps/publisher/site/public/dist && cp -R /home/tharani/clone/apim-apps/portals/publisher/site/public/dist /home/tharani/clone/pack/repository/deployment/server/jaggeryapps/publisher/site/public ' "
```
Comment (madushadhanushka) [inline portals/publisher/package.json:23]: Replace script in here as well. lets remove it

File: portals/devportal/site/public/locales/en.json [hunk 3]
```diff
@@ -409,10 +409,10 @@
   "Base.Errors.ResourceNotFound.more.links": "You may check the links below",
   "Base.Errors.ResourceNotfound.default_body": "The page you are looking for is not available",
   "Base.Errors.ResourceNotfound.default_tittle": "Page Not Found",
-  "Base.Errors.SubscriptionNotFound.default_title": "Solace Info Page is not displayed without subscriptions to the API. Please Subscribed to the API",
   "Base.Errors.ScopeNotFound.message.first": "Sorry, the page you are looking for ",
   "Base.Errors.ScopeNotFound.message.second": "is not allowed under logged in user role scopes. Please login with different user with relevant permission to access this resource.",
   "Base.Errors.ScopeNotFound.title": "Un-authorized Access",
+  "Base.Errors.SubscriptionNotFound.default_title": "Solace Info Page is not displayed without subscriptions to the API.\n                                        Please Subscribed to the API",
```
Comment (madushadhanushka) [inline portals/devportal/site/public/locales/en.json:415]: Shall we remove this change if not required

Comment (Tharanidk) [reply portals/devportal/site/public/locales/en.json]: Done Changes.

**Review by madushadhanushka — APPROVED** [review]

**Pull request merged**